main.go

```diff
@@ -9,8 +11,24 @@ import (
 )
 
 func main() {
+	var debugMode bool
+
 	log.Println("IBM Cloud Provider version", version.Version, version.VersionPrerelease, version.GitCommit)
-	plugin.Serve(&plugin.ServeOpts{
+
+	flag.BoolVar(&debugMode, "debug", false, "set to true to run the IBM terraform provider with support for debuggers like delve")
+	flag.Parse()
+
+	opts := &plugin.ServeOpts{
 		ProviderFunc: ibm.Provider,
-	})
+	}
+
+	if debugMode {
+		err := plugin.Debug(context.Background(), "IBM-Cloud/ibm", opts)
```

Comment:
should it not be `registry.terraform.io/IBM-Cloud/ibm` instead of `IBM-Cloud/ibm`?

Reply:
I tested with `IBM-Cloud/ibm` but I can test with full url too

Reply:
Looks like it works both ways, so I changed to full url